schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -484,6 +479,7 @@
             }
         },
         "adpContainer": {
+            "$comment": "TODO: description needed",
```

Comment:
An object containing the vulnerability information provided by an Authorized Data Publisher (ADP).  Since multiple ADPs can provide information for a CVE ID, the adpContainer must indicate which ADP is the source of the information in the object.